type: experiment/online_learning
rate: 0
environment:
  type: environment/modeled
  model:
    type: model/dynamical
    dynamics:
      type: dynamics/pendulum
    control_step: 0.03
  task:
    type: task/pendulum/swingup
agent:
  type: agent/ros
  title: mops_sim
  observation_min: environment/task/observation_min
  observation_max: environment/task/observation_max
  action_min: environment/task/action_min
  action_max: environment/task/action_max
  reward_min: -50
  reward_max: 0
  stochastic: 0
  episodic: 1
